x2gbfs/x2gbfs.py

```diff
@@ -54,12 +54,16 @@ def build_extractor(provider: str, feed_config: Dict[str, Any]) -> BaseProvider:
         or provider.startswith('teilauto_')
     ):
         return CantamenIXSIProvider(feed_config)
-    if provider in ['stadtwerk_tauberfranken', 'zeag_energie', 'flinkster_carsharing']:
+    if provider in [
+        'stadtwerk_tauberfranken',
+        'zeag_energie',
+        'flinkster_carsharing',
+        'oberschwabenmobil',
+        'gmuend_bewegt',
+    ]:
         return MoqoProvider(feed_config)
     if provider in ['noi']:
         return NoiProvider()
-    if provider in ['flinkster']:
-        return FlinksterProvider(feed_config)
```

Comment:
An oversight in the review: you removed the `flinkster` option here, but kept the import, the config and the `FlinksterProvider` itself. Either it should be removed completely or kept here as well.